api_formatter/static/js/custom.js

```diff
@@ -1,3 +1,15 @@
-$(document).ready(function() {
-    $('form').ajaxForm();
-});
\ No newline at end of file
+// Handles clicks on the mobile nav toggle button
+document.getElementById('nav-toggle').addEventListener('click', function() {
+    const expandedCurrent = this.getAttribute('aria-expanded')
+    
+    // Sets classes and attributes on the nav-toggle
+    this.classList.toggle('active')
+    this.classList.toggle('closed')
+    this.setAttribute('aria-expanded', expandedCurrent === 'true' ? 'false' : 'true')
+
+    // Sets classes and attributes on the nav-toggle-menu
+    menu = document.getElementById('nav-toggle-menu')
+    menu.classList.toggle('active')
+    menu.classList.toggle('closed')
+    menu.setAttribute('aria-expanded', expandedCurrent === 'true' ? 'false' : 'true')
```

Comment:
I don't think we need this `aria-expanded` for the `ul`, only for the toggle button